.github/workflows/pipeline-deploy-app-develop.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
   deploy-application-infrastructure:
       needs:
         [
-          unit-tests,
+          # unit-tests,
```

Comment:
Remove unit tests from deployment dependency. Might need to deploy in initial stages of dv